salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -20,5 +20,23 @@ edx:
         roles:
           - edx
           - edx-{{ app_type }}
+
+  - edx-worker-{{ environment_name }}-{{ app_type }}-{{ id_num }}:
+      network_interfaces:
+        - DeviceIndex: 0
+          AssociatePublicIpAddress: False
```

Comment:
This needs to be `True` or else the instance won't be able to access the internet to download dependencies.